source/core/parameters.cc

```diff
@@ -1531,6 +1531,11 @@ namespace Parameters
   {
     prm.enter_subsection("mesh adaptation");
     {
+      prm.declare_entry("initial refinement steps",
```

Comment:
Number of initial refinement step would be better I think

Reply:
I think a little bit long for a parameter name